doc/Developer-Guide.md

```diff
@@ -1,38 +1,92 @@
 
-*Basic configurations* are declared using YAML, hereto: [Configuration file](Configuration-file.md). The main configuration file is [default.yml](https://github.com/tuub/kitodo-mediaserver/blob/master/kitodo-mediaserver-core/src/main/resources/config/default.yml). Any configuration parameter in this file can be overridden in [local.yml](https://github.com/tuub/kitodo-mediaserver/blob/master/kitodo-mediaserver-core/src/main/resources/config/local.yml). The latter must in the repository be left empty, only to be used for local installations. Occasionally there are additional configurations in the application.yml of each module.
+The local module is the place where to put local configurations and extensions and is thus empty in the main repository.
+
+## Configuration files
 
-The read order of the configuration files is set in the <module>Application class of each module. The embedding of the configuration parameter in the java code takes place in @ConfigurationProperties classes in the config directories of the modules. If you add a configuration parameter, you also have to extend the appropriate @ConfigurationProperties class. 
+*Basic configurations* are declared using YAML, hereto: [Configuration file](Configuration-file.md). The main configuration file is [default.yml](../kitodo-mediaserver-core/src/main/resources/config/default.yml). Any configuration parameter in this file can be overridden in [local.yml](../kitodo-mediaserver-local/src/main/resources/config/local.yml). The latter must in the repository be left empty, only to be used for local installations. Occasionally there are additional configurations in the application.yml of each module.
+
+The system also allows a dev.yml to be used for developing puposes and a secrets.yml containing passwords and suchs to be deployed only on production servers. An example of a dev.yml using console logging and hibernate printouts looks like this:
+```yml
+spring:
+  jpa:
+    properties:
+      hibernate:
+        show_sql: true
+        use_sql_comments: true
+        format_sql: true
+
+logging:
+  path: /srv/kitodo/mediaserver/logs/
+  level:
+    org.springframework: INFO
+    com.zaxxer.hikari: INFO
+    org.hibernate: INFO
+    org.kitodo: DEBUG
+  pattern:
+    console: "%d{yyyy-MM-dd HH:mm:ss:SSS} [%thread] %-5level %logger - %msg%n"
+```
 
-Note that the yaml files also can be overridden by external files at execution, see TODO.
+The read order of the configuration files is set in the `@SpringBootApplication class of each executable module, see e.g. [FileserverApplication.java](kitodo-mediaserver-fileserver/src/main/java/org/kitodo/mediaserver/fileserver/FileserverApplication.java). The embedding of the configuration parameter in the java code takes place in `@ConfigurationProperties` classes in the config directories of the modules. If you add a configuration parameter, you also have to extend the appropriate `@ConfigurationProperties` class.
```

Comment:
The path should be `../kitodo-mediaserver-fileserver`... I think (did not test it)